server/lib/src/idm/authsession.rs

```diff
@@ -714,6 +714,15 @@ impl AuthSessionState {
     }
 }
 
+pub(crate) struct AuthSessionData<'a> {
```

Comment:
This isn't just a clippy/fmt change and should probably be out in `kanidm_proto`

Reply:
This would be an internal struct too, not proto.